### PR TITLE
V3 fixes

### PR DESCRIPTION
**Pull request by kevinvangelder**

### CODE REVIEW

File: src/extensions/functions/onboard/kickoffs/kickoffCLI.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import { SolidarityRunContext } from '../../../../types'
+import { SolidarityRunContext, SolidarityRule } from '../../../../types'
 
-export default async (context: SolidarityRunContext): Promise<void> => {
-  const { print } = context
-  print.info(`kickoff CLI function`)
-  // return { rule: 'cli' }
+export default async (context: SolidarityRunContext): Promise<SolidarityRule> => {
+  // const { print } = context
+  // print.info(`kickoff CLI function`)
```
Comment (gwmccull) [inline src/extensions/functions/onboard/kickoffs/kickoffCLI.ts]: is this no longer needed?

File: src/extensions/functions/onboard/kickoffs/kickoffShell.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
-import { SolidarityRunContext } from '../../../../types'
+import { SolidarityRunContext, SolidarityRule } from '../../../../types'
 
-export default async (context: SolidarityRunContext): Promise<void> => {
-  const { print } = context
-  print.info(`kickoff Shell function`)
+export default async (context: SolidarityRunContext): Promise<SolidarityRule> => {
+  // const { print } = context
+  // print.info(`kickoff Shell function`)
```
Comment (gwmccull) [inline src/extensions/functions/onboard/kickoffs/kickoffShell.ts]: no longer needed?

**Comment by gwmccull** [comment]
@kevinvangelder if you want, once you're all in master, I can try it out on our project

File: src/extensions/functions/onboard/kickoffs/kickoffCLI.ts (hunk 1 above)
Comment (kevinvangelder) [reply src/extensions/functions/onboard/kickoffs/kickoffCLI.ts]: No, I believe the print statement was just there for debugging while implementing this feature.

**Comment by kevinvangelder** [comment]
@gwmccull Yes, I did the package.json changes on master (yeah, I know).

**Pull request merged**